db/schema.rb

```diff
@@ -137,17 +137,17 @@
 
   create_table "course_assessments", force: :cascade do |t|
     t.integer  "tab_id",                    :null=>false, :index=>{:name=>"fk__course_assessments_tab_id"}, :foreign_key=>{:references=>"course_assessment_tabs", :name=>"fk_course_assessments_tab_id", :on_update=>:no_action, :on_delete=>:no_action}
-    t.boolean  "tabbed_view",               :default=>false, :null=>false
     t.boolean  "autograded",                :null=>false
-    t.boolean  "show_private",              :default=>false, :comment=>"Show private test cases  after students answer correctly"
-    t.boolean  "show_evaluation",           :default=>false, :comment=>"Show evaluation test cases after after students answer correctly"
-    t.boolean  "skippable",                 :default=>false
-    t.boolean  "delayed_grade_publication", :default=>false
-    t.string   "password",                  :limit=>255
     t.integer  "creator_id",                :null=>false, :index=>{:name=>"fk__course_assessments_creator_id"}, :foreign_key=>{:references=>"users", :name=>"fk_course_assessments_creator_id", :on_update=>:no_action, :on_delete=>:no_action}
     t.integer  "updater_id",                :null=>false, :index=>{:name=>"fk__course_assessments_updater_id"}, :foreign_key=>{:references=>"users", :name=>"fk_course_assessments_updater_id", :on_update=>:no_action, :on_delete=>:no_action}
     t.datetime "created_at",                :null=>false
     t.datetime "updated_at",                :null=>false
+    t.string   "password",                  :limit=>255
+    t.boolean  "tabbed_view",               :default=>false, :null=>false
+    t.boolean  "skippable",                 :default=>false
+    t.boolean  "delayed_grade_publication", :default=>false
```

Comment:
These columns should be touched. You can reset the db before running the migration.